README.md

```diff
@@ -47,7 +47,9 @@ module "tf_infra_pipeline" {
 | emails | List of email-addresses receiving notifications on updates | list(string) | [] | All recipient will receive confirmation email from AWS |
 | failure_notifications | Whether or not you want notifications on failed builds | bool | true |  |
 | success_notifications | Whether or not you want notifications on succeeded builds | bool | false |  |
-
+| checkov_skip_checks | List of CheckOV check you want to skip | string | "" |
+| directory | directory for terraform hcl | string | "/" |
+| extra_build_artifacts | filename to include for codepipeline apply step | string | "" |
```

Comment:
I was thinking if this should be `set(string)`. It would be easier to export multiple artifacts, not just one more

Reply:
Could be more useful in the future, but then needs to fix template to create correct format in the output plan file